.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -755,7 +755,7 @@ jobs:
           OSSRH_PASSWORD: ${{ secrets.sdk-ossrh-password }}
         with:
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: ":swirlds-platform:release{{ inputs.sdk-release-profile }} --scan -PpublishSigningEnabled=true"
+          arguments: ":platform-sdk:release${{ inputs.sdk-release-profile }} --scan -PpublishSigningEnabled=true"
```

Comment:
Note that instead of doing this, we should really do this: https://github.com/hashgraph/hedera-services/pull/8529
(once this is merged we have to adjust the other PR to change it back)

Reply:
I changed the approach in #8529 (call in "platform-sdk" consistently) it now contains the same change as here and should be compatible.